xbmc/guilib/GUIFontTTF.h

```diff
@@ -68,29 +67,45 @@ struct SVertex
 #include "GUIFontCache.h"
 
 
-class CGUIFontTTF
+class CGUIFontTTF : public std::enable_shared_from_this<CGUIFontTTF>
 {
+  static constexpr size_t LOOKUPTABLE_SIZE = 256 * 8;
+  static constexpr int CHARS_PER_TEXTURE_LINE = 20; // number characters to cache per texture line
```

Comment:
These should go into the cpp file in an anonymous namespace and they don't need to be static